src/deltaE/deltaE2000.js

```diff
@@ -81,13 +81,12 @@ Color.prototype.deltaE2000 = function (sample, {kL = 1, kC = 1, kH = 1} = {}) {
 	// Hue difference, getting the sign correct
 	let hdiff = h2 - h1;
 	let hsum = h1 + h2;
-	let habs = Math.abs(hdiff);
 	let Δh;
 
-	if (Cdash1 == 0 && Cdash2 == 0) {
+	if (Cdash1 * Cdash2 == 0) {
```

Comment:
Is there a reason not to use `===` here? (question to both you and @svgeesus )

Reply:
No, I didn't use `===` because I write in many languages and accidentally didn't use JS `===`.